packages/muelu/src/Utils/ForwardDeclaration/MueLu_RegionRFactory_fwd.hpp

```diff
@@ -46,8 +46,8 @@
 #ifndef MUELU_REGIONRFACTORY_FWD_HPP
 #define MUELU_REGIONRFACTORY_FWD_HPP
 
-
-
+#include "MueLu_ConfigDefs.hpp"
+#if defined(HAVE_MUELU_KOKKOS_REFACTOR)
```

Comment:
Do we need the Kokkos refactor branch for that or just Tpetra to be enabled?

Reply:
`HAVE_MUELU_TPETRA` seems to suffice.

Reply:
Again definitely need Kokkos for this.

Reply:
Will change this back.